app/src/contexts/auth.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { createContext, useEffect, useState } from 'react'
+
+import app from 'STARTER/feathers-client'
+
+export const AuthContext = createContext({})
```

Comment:
I've been thinking a bit more about how to improve this file, especially now that it will be the starting point for future projects.

When we call createContext we can provide the AuthContextType `createContext<AuthContextType>({})`. This might help so that we don't always need to do `useContext(AuthContext) as AuthContext`.

I also think it will help to name the default export. In devtools it shows up as `__WEBPACK_DEFAULT_EXPORT__`, which isn't super helpful. So maybe we could do something like 
```Typescript
const AuthProvider = ({ children: any }) => {
...
}

export default AuthProvider
```

I've still been trying to think of a way to try to prevent the app from rendering three times at the start, but I've not come up with anything great.

Reply:
I'll look into implementing this and let you know if I have any questions in the process, thanks so much!

Reply:
okay I pushed a commit that I think addresses this comment but let me know if I missed / misunderstood anything!

Reply:
I thought I commented on this yesterday, but don't see the comment now. Thanks so much for making these changes :) This looks great!